lib/mixlib/install/cli.rb

```diff
@@ -32,7 +33,9 @@ def list_versions(product_name, channel)
       option :platform_version,
         aliases: ["-l"]
       option :architecture,
-        aliases: ["-a"]
+        default: "x86_64",
+        aliases: ["-a"],
+        enum: %w{i386 s390x sparc x86_64}
```

Comment:
Technically all of them are "correct", any of the the 3 should return the non-sparc packages.